src/vorta/views/archive_tab.py

```diff
@@ -365,14 +367,18 @@ def on_selection_change(self, selected=None, deselected=None):
             reason = reason or self.tr("(Select exactly one archive)")
 
             # too few or too many selected.
-            self.fArchiveActions.setEnabled(False)
+            self.bMountArchive.setEnabled(False)
+            self.bExtract.setEnabled(False)
+            self.bRename.setEnabled(False)
 
             for index in range(layout.count()):
                 widget = layout.itemAt(index).widget()
                 tooltip = widget.toolTip()
 
                 tooltip = self.tooltip_dict.setdefault(widget, tooltip)
-                widget.setToolTip(tooltip + " " + reason)
+
+                if widget != self.bRefreshArchive:
+                    widget.setToolTip(tooltip + " " + reason)
```

Comment:
Because every button is disabled when multiple archives are selected except the refresh button. The "reason" variable here is - Only one archive should be selected (similar).

Reply:
What about:

```
 # too few or too many selected.
for widget in [self.fArchiveActions, self.bMountArchive, ...]:
    tooltip = widget.toolTip()
    tooltip = self.tooltip_dict.setdefault(widget, tooltip)
    widget.setToolTip(tooltip + " " + reason)
    widget.setEnabled(False)

# special treatment for dynamic mount/unmount button.
self.bmountarchive_refresh()
tooltip = self.bMountArchive.toolTip()
self.bMountArchive.setToolTip(tooltip + " " + reason) 

```

Reply:
In understand, but that also applies for the diff button. Can't you treat them similarly?

Reply:
> What about:
> 
> ```python
> # too few or too many selected.
> for widget in [self.fArchiveActions, self.bMountArchive, ...]:
>     tooltip = widget.toolTip()
>     tooltip = self.tooltip_dict.setdefault(widget, tooltip)
>     widget.setToolTip(tooltip + " " + reason)
>     widget.setEnabled(False)
> 
> # special treatment for dynamic mount/unmount button.
> self.bmountarchive_refresh()
> tooltip = self.bMountArchive.toolTip()
> self.bMountArchive.setToolTip(tooltip + " " + reason) 
> ```

self.fArchiveActions will include the refresh button though, so this snippet did not work.
But with the same idea, this works:

```python
        if self.repoactions_enabled and len(indexes) == 1:
            # Enable archive actions
            for widget in [self.bMountArchive, self.bExtract, self.bRename]:
                widget.setEnabled(True)

            for index in range(layout.count()):
                widget = layout.itemAt(index).widget()
                widget.setToolTip(self.tooltip_dict.get(widget, ""))

            # refresh bMountArchive for the selected archive
            self.bmountarchive_refresh()
        else:
            reason = reason or self.tr("(Select exactly one archive)")

            # too few or too many selected.
            for widget in [self.bMountArchive, self.bExtract, self.bRename]:
                tooltip = widget.toolTip()
                tooltip = self.tooltip_dict.setdefault(widget, tooltip)
                widget.setToolTip(tooltip + " " + reason)
                widget.setEnabled(False)
```